proto/batch_plan.proto

```diff
@@ -99,6 +100,19 @@ message FileScanNode {
   repeated string file_location = 7;
 }
 
+message GcsFileScanNode {
+  enum FileFormat {
+    FILE_FORMAT_UNSPECIFIED = 0;
+    PARQUET = 1;
+  }
+
+  repeated plan_common.ColumnDesc columns = 1;
+  FileFormat file_format = 2;
+  string credential = 3;
+  string service_account = 4;
```

Comment:
Why we need 2 fields here stand for? IIRC, we only need a service account json encoded by base64.

Reply:
see [previously discussion](https://github.com/risingwavelabs/risingwave/pull/17311#discussion_r1701289670)

Reply:
After some offline discussion, we'd like to remove `service_account` for both iceberg sink and file scan.